### ISSUE TITLE
positive i64s round trip as u64s (obv, cuz we dont have type info): is this what we want?

### ISSUE THREAD

**Issue by phritz** [rocicorp/repc#141]
https://github.com/rocicorp/repc/pull/140

i noticed this when one of the tests i'm writing failed for this reason (the expected value's type was i64 but we got a u64 back from round trip)

**Comment by aboodman** [comment]
Right. Maybe the right thing is to remove the signed variants from `Any`? And `f64` for that matter?

**Comment by phritz** [comment]
I think you meant to say remove the unsigned variants, but I think so.

**Comment by aboodman** [comment]
Right basically anything we can't roundtrip.

**Comment by aboodman** [comment]
I think this should be fixed by just not treating the content as JSON at all: #139, #162.

**Comment by phritz** [comment]
Righ, but isn't the issue that would eliminate Any and thus obviate this issue https://github.com/rocicorp/repc/issues/141?
